src/components/Accordion/Accordion.tsx

```diff
@@ -18,15 +18,21 @@ type AccordionProps = {
    */
   className?: string;
   /**
-   * Outline variant adds extra padding and a rounded border.
+   * Outline variant adds adjusts the `Accordion` style by defining a containing border and other layout adjustments.
+   *
+   * **Default is `false`.**
    */
   hasOutline?: boolean;
   /**
-   * Used to specify which heading element should be rendered for each Accordion.Title child.
+   * Used to specify which heading element should be rendered for each `Accordion.Title` child.
+   *
+   * **Default is `"h2"`**.
```

Comment:
Convention: Call out the default value in prop doc blocks, so that it shows up both in storybook as highlighted, and in the IDE.